panels/UserPanel.php

```diff
@@ -116,4 +124,19 @@ protected function dataToString($data)
 
         return VarDumper::export($data);
     }
+
+    /**
+     * Returns the array/object that should be set on [[\yii\widgets\DetailView::model]]
+     *
+     * @param IdentityInterface $identity
+     * @return array|object
+     */
```

Comment:
it's useful for easier understanding of the method and this will also let us benefit from the PHP7 return typing